ui/client/components/tips/Errors.js

```diff
@@ -65,20 +68,20 @@ export default class Errors extends React.Component {
         <div className={"node-error-links"}>
           <NodeErrorsLinkSection
             nodeIds={_.concat(otherNodeErrorIds, _.isEmpty(propertiesErrors) ? [] : "properties")}
-            message={"Errors in: "}
+            message={i18next.t("errors.errorsIn", "Errors in: ")}
```

Comment:
better move to FC (and tsx) with hook

Reply:
In next PR, this one should be small to be merged into 0.4.0, and for now we don't have >1 language version anyway...